src/com/nutrons/framework/subsystems/WpiSmartDashboard.java

```diff
@@ -0,0 +1,32 @@
+package com.nutrons.framework.subsystems;
```

Comment:
Add a TODO comment for us to implement the remaining functionality.  There's a bunch more stuff in the Java class and we might want to have it during competitions.